SPECS/rt-tests/rt-tests.spec

```diff
@@ -102,8 +76,12 @@ make %{?_smp_mflags} DESTDIR=%{buildroot} prefix=%{_prefix} install
 /usr/share/man/man8/ssdd.8.gz
 /usr/share/man/man8/oslat.8.gz
 /usr/share/man/man8/get_cyclictest_snapshot.8.gz
+/usr/share/man/man8/determine_maximum_mpps.8.gz
 
 %changelog
```

Comment:
Also, since it looks like you made some manual modifications here, you might want to add your name to the CL instead of cbl-mariner Servicing Account/cblmargh@microsoft.com